client/components/Medical-Contact/ContactPage.vue

```diff
@@ -0,0 +1,51 @@
+<!-- Page for medical contacts management -->
+<!-- User should be authenticated in order to see this page -->
+
+<template>
+  <main>
+    <hr>
+    <section>
+      <header>
+        <h2>Health Book</h2>
+      </header>
+      <!-- <button>
+        Add contact
+      </button> -->
+      <CreateContactForm />
+    </section>
+    <hr>
+    <section>
```

Comment:
This section should have a header. It might be a bit unintuitive currently what the section is for someone who has never seen the app before